src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/RazorDisableDropHandlerProvider.cs

```diff
@@ -9,14 +9,17 @@
 using Microsoft.VisualStudio.Text.Editor.DragDrop;
 using Microsoft.VisualStudio.Utilities;
 
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
+    // The intention of this class is to disable dropping random files into the Razor language service content type without throwing. Ultimately
```

Comment:
Super helpful comment! When I read the PR title my first question was what drop handling was